bls-keystore/src/main/java/tech/pegasys/signers/bls/keystore/model/KeyStoreData.java

```diff
@@ -38,18 +38,18 @@
   public KeyStoreData(
```

Comment:
you may want a INCLUDE_NON_NULL - or whatever it is, such that when we generate a file, we don't put in null fields (i.e. the resulting json has the field missing, rather than field: null)

Reply:
for generation this constructor is not used. But you are right, let me add that annotation.